### DISCUSSION TITLE
Top level routes "" and "/" are treated as two distinct routes

### DISCUSSION THREAD

**Discussion by Mardoxx** [fastapi/fastapi#9412] [Questions]
### Issue Content

Title. See also:

https://github.com/encode/httpx/issues/2673

One of you is correct. It can't be both.

**Comment by Mardoxx** [comment]
This means if you configure:

@get("")
def route() return True

and curl to "/"

You'll get a 307 instead of a 200.

**Comment by mateoradman** [comment]
As this person explained https://github.com/encode/httpx/issues/2673#issuecomment-1515353672, 

> Note that the absolute path
   cannot be empty; if none is present in the original URI, it MUST be
   given as "/" (the server root).
